getting_started/step_by_step/your_first_game.rst

```diff
@@ -267,16 +336,32 @@ Let's place this code at the end of our ``_process()`` function:
         elif velocity.y != 0:
             $AnimatedSprite.animation = "up"
             $AnimatedSprite.flip_v = velocity.y > 0
+ 
+ .. code-tab:: csharp
+
+        if(velocity.x != 0) {
+            animated_sprite.Animation = "right";
+            animated_sprite.FlipH = velocity.x < 0;
+            animated_sprite.FlipV = false;
+        } else if(velocity.y != 0) {
+            animated_sprite.Animation = "up";
+            animated_sprite.FlipV = velocity.y > 0;
+        }
 
 Play the scene again and check that the animations are correct in each
 of the directions. When you're sure the movement is working correctly,
 add this line to ``_ready()`` so the player will be hidden when the game
 starts:
 
-::
+.. tabs::
+ .. code-tab:: gdscript GDScript
     
     hide()
 
+ .. code-tab:: csharp
+
+    this.Hide();
```

Comment:
I'm not too sure about this, but my IDE complains if I use `this.` when it's not an extention method call.